benchmarks/infra/stage-1/sample-tfvars/jetstream-sample.tfvars

```diff
@@ -0,0 +1,27 @@
+project_id   = "PROJECT_ID"
+cluster_name = "ai-tpu-benchmark"
+region       = "us-east1"
+gke_location = "us-east1-c"
+prefix       = "ai-tpu-benchmark"
+spot_vms     = true
+
+vpc_create = {
+  name             = "ai-benchmark"
```

Comment:
adding a TODO on the stage 1 infra where the cluster name is defined

Reply:
ty!

Reply:
Calling the cluster name just ai-benchmark makes sense since we wouldn't have to change the fleet host everywhere to GPU and TPU separately. I'm inclined to keeping this ai-benchmark as well to keep it simple.